src/pages/customers/AddNewCustomers.tsx

```
@@ -47,7 +47,7 @@ const AddNewCustomers: React.FC = () => {
       };
 
     const { mutate, isLoading: isSubmitting } = useMutation({
-        mutationFn: (values: any) => addCustomer(values, authHeader()),
+        mutationFn: (values: any) => addCustomer(values, useAuthToken() || ""),
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)